v2/README.md

```diff
@@ -59,14 +59,19 @@ It's important for renters and hosts to upgrade before **June 6th** so that they
-If you do NOT have a synced Sia-UI instance, then you can log into SiaCentral with a 28/29-word seed and transfer coins that way, too. SiaCentral will continue to support the "old" Sia seed format along side the new one, but official apps will only support the new 12-word seed format.
+`walletd` only supports the new 12-word seed format and does **not** support legacy 28- or 29-word seeds. To ensure continued access to your coins, we **strongly recommend** generating a new 12-word seed and transferring your balance. Use the [SiaCentral Web Wallet](https://wallet.siacentral.com/) to create your new seed and Deposit Address.
 
-We recommend that Siacoin holders upgrade to `walletd` by **June 6th**. If you don't, you'll still be able to broadcast transactions during the 1-month grace period, but you won't be able to sync new blocks. In order to process transactions after **July 4th**, you must use software that supports signing v2 transactions. Currently, that is only `walletd` and SiaCentral's Web Wallet; Sia-UI will not be updated.
+- **If you're using a synced Sia-UI wallet:**  
+  First send a small test amount (~10 SC) to the new address to confirm it’s received. Then transfer the rest, leaving 1 SC behind if needed to cover the network fee.
 
-If you store Siacoin on an exchange, ensure that your exchange has committed to supporting the fork. We will be reaching out to exchanges to notify them of the fork and offer support, but ultimately the exchange is responsible for updating their nodes and ensuring that you can continue to deposit and withdraw Siacoins. To be sure that your tokens will be accessible after the fork, we **strongly recommend** withdrawing them to a privately owned address as we are not responsible for how exchanges handle tokens.
+- **If you don’t have a synced Sia-UI wallet:**  
+  Log into SiaCentral with your 28/29-word seed and transfer your coins from there. SiaCentral will continue to support both old and new seed formats, but official apps will only support the new 12-word format.
+
+We recommend all Siacoin holders upgrade to `walletd` by **June 6th**. After that, you'll have a 1-month grace period to broadcast transactions, but syncing new blocks won't be possible. Starting **July 4th**, only software that supports v2 transaction signing—currently `walletd` and the SiaCentral Web Wallet—will work. Sia-UI will not be updated.
+
+If you store Siacoins on an exchange, confirm that the exchange will support the fork. While we’ll notify and assist exchanges, it’s their responsibility to upgrade their nodes. To ensure access to your tokens, we **strongly recommend** withdrawing them to a privately owned wallet before the fork.
```

Comment:
Remove this: `While we’ll notify and assist exchanges, it’s their responsibility to upgrade their nodes.`